chainercv/links/model/segnet/segnet_basic.py

```diff
@@ -0,0 +1,81 @@
+import chainer
+import chainer.functions as F
+import chainer.links as L
+
+
+class SegNetBasic(chainer.Chain):
+
+    """SegNet Basic for semantic segmentation.
+
+    This is a SegNet [#]_ model for semantic segmenation. This is based on
+    SegNetBasic model that is found here_.
+
+    .. [#] Vijay Badrinarayanan, Alex Kendall and Roberto Cipolla "SegNet: A \
+    Deep Convolutional Encoder-Decoder Architecture for Image Segmentation." \
+    PAMI, 2017 
+
+    .. _here: http://github.com/alexgkendall/SegNet-Tutorial
+
+    Args:
+        out_ch (int): The number of channels for the final convolutional layer.
```

Comment:
It seems that Chainer's links use `out_channel` across the library.
Why not use `out_channel` instead of `out_ch`?